identity-service.js

```diff
@@ -9,17 +9,39 @@ define([], function() {
 
 'use strict';
 
+function register(module) {
+  module.service('brIdentityService', factory);
+}
+
 /* @ngInject */
 function factory(
-  $rootScope, $routeParams, brRefreshService, brResourceService, config) {
+  $rootScope, $routeParams, brSessionService,
+  brRefreshService, brResourceService, config) {
   var service = {};
+  var config = config.data.identity;
+  service.basePath = config.baseUri;
 
-  var identity = config.data.identity;
   service.collection = new brResourceService.Collection({
-    url: identity.baseUri
+    url: config.baseUri
   });
   service.state = service.collection.state;
 
+  // Gets the currently authenticated identity
+  service.get = function() {
+    // Grab the identity from the session
+    if(!brSessionService.session || !brSessionService.session.identity) {
+      return Promise.reject(new Error(
+        'No session available')
+      );
+    }
+    var identity = brSessionService.session.identity;
+    return service.collection.get(identity.id)
```

Comment:
This will break if the identity id isn't a full URL (and we don't necessarily force it to be) -- and an identity's sysSlug isn't available here because the session doesn't track it (bedrock-identity-http accepts routes to its collection that have a sysSlug appended, i.e. /i/testName).